backend/apps/profile_info/api.py

```diff
@@ -0,0 +1,84 @@
+
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status, serializers
+from rest_framework.permissions import IsAuthenticated
+from rest_framework_simplejwt.authentication import JWTAuthentication
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+
+from apps.users.models import BaseUser
+from apps.profile_info.services import update_user_profile
+from apps.profile_info.selectors import get_user_profile
+
+class ProfileInfoView(APIView):
```

Comment:
Split this into CRUD operations instead of a single view - and any business logic shouldn't take place in the views. See community apis for examples.